packages/world-plugins/.nvmrc

```diff
@@ -0,0 +1 @@
+17
```

Comment:
we're moving to 18 in https://github.com/latticexyz/mud/pull/578, should account for that here already